lib/commands.py

```diff
@@ -240,6 +240,14 @@ def is_command(node):
     return CONNECTION_START in node.attrib or CONNECTION_END in node.attrib
 
 
+def is_command_symbol(node):
+    symbol = None
+    xlink = node.get(XLINK_HREF, "")
+    if len(xlink) > 11:
```

Comment:
Maybe `xlink.startswith("#inkstitch_")` here so that it's more clear what you're doing?